lib/extended_button.dart

```diff
@@ -186,24 +191,24 @@ class _ExtendedButtonState extends State<ExtendedButton> {
 
 class SquareBox extends StatefulWidget {
   final IconData icon;
-  final bool opened;
+  final bool opened, isRounded;
```

Comment:
You are never passing the `widget.isRounded` to `SquareBox`. That's why test cases are failing.